### PR TITLE
More news buttons wiring and Story details

### PR DESCRIPTION
**Pull request by akintolga**
- They will appear if the company has access to that product
- Also added class name to notifications

### CODE REVIEW

File: assets/components/NotificationList.jsx [hunk 1]
```diff
@@ -16,10 +16,16 @@ class NotificationList extends React.Component {
...
+        if (!this.state.displayItems) {
+            document.getElementById('header-notification').classList.add('notif--open');
+        } else {
+            document.getElementById('header-notification').classList.remove('notif--open');
+        }
     }
 
     renderNotification(newItem) {
         return (<div key={newItem._id} className='notif__list__item m-2 p-3'>
+            <p>A story you downloaded has been updated</p>
```
Comment (petrjasek) [inline assets/components/NotificationList.jsx]: gettext missing

File: assets/home/components/TextOnlyCard.jsx [hunk 2]
```diff
@@ -21,14 +21,15 @@ const getTextOnlyPanel = item => (
...
+function TextOnlyCard({items, title, product}) {
     return (
         <div className='row'>
             <div className='col-6 col-sm-8'>
                 <h3 className='home-section-heading'>{title}</h3>
             </div>
             <div className='col-6 col-sm-4 d-flex align-items-start justify-content-end'>
-                <button type='button' className='btn btn-outline-primary btn-sm mb-3'>{gettext('More news')}</button>
+                {product &&
+                <button onClick={() => window.location.href = `/wire?q=${getProductQuery(product)}`} type='button' className='btn btn-outline-primary btn-sm mb-3'>{gettext('More news')}</button>}
```
Comment (petrjasek) [inline assets/home/components/TextOnlyCard.jsx]: looks like a link

File: assets/home/components/TopNewsOneByOneCard.jsx [hunk 3]
```diff
@@ -41,14 +41,15 @@ const getTopNewsPanel = (item, picture) => {
...
+function TopNewsOneByOneCard({items, title, product}) {
     return (
         <div className='row'>
             <div className='col-6 col-sm-8'>
                 <h3 className='home-section-heading'>{title}</h3>
             </div>
             <div className='col-6 col-sm-4 d-flex align-items-start justify-content-end'>
-                <button type='button' className='btn btn-outline-primary btn-sm mb-3'>{gettext('More news')}</button>
+                {product &&
+                <button onClick={() => window.location.href = `/wire?q=${getProductQuery(product)}`} type='button' className='btn btn-outline-primary btn-sm mb-3'>{gettext('More news')}</button>}
```
Comment (petrjasek) [inline assets/home/components/TopNewsOneByOneCard.jsx]: same here, actually looks like this could be a component it's duplicated in a few cards already

**Comment by akintolga** [comment]
@petrjasek Fyi, the last commit in this PR contains changes for the story details pop-up from home page.

**Pull request merged**